programs/eosio-launcher/main.cpp

```diff
@@ -849,6 +850,7 @@ launcher_def::define_network () {
     } // for i
     bindings.emplace_back( move(*lhost) );
     delete lhost;
+    lhost = nullptr;
```

Comment:
Since this is a local variable, I don't think this is needed, but if we make this change, I think for maintainability we should instead make lhost a unique_ptr and change line 810 and 851 to be "bindings.push_back(*lhost);" so we don't have to keep track of memory. This use case will not loop through this enough times for the moving of memory to make a significant impact.

Reply:
Brian - please take a look at the revised implementation. Thanks.